src/www/ui_components/server-card.html

```diff
@@ -169,6 +177,10 @@
     Polymer({
       is: "server-card",
       properties: {
+        SUPPORTED_CIPHERS: {
```

Comment:
The UI element shouldn't know about what ciphers are supported.
Instead, expose only isCipherSupported, and let the caller set that.

Provide a method isCipherSupported(cipher), rather than a list. It's more flexible and extensible. The caller can use that method to set the field here.

That will also remove the duplicate list.

Reply:
That makes sense.

The code duplication is bad. I chose this way because I don't know of how to share the logic between Typescript and Polymer 2 easily. Where could define it so that I can use it in both places?

Reply:
As discussed on chat, the cipher validation shouldn't be in the UI. It should be in the Typescript application code. Let's move it there. The UI should just know how to display an error and disable connection.

Reply:
Thanks for the suggestion! Done.